modules/nw-networkpolicy-multitenant-isolation.adoc

```diff
@@ -94,56 +89,44 @@ spec:
 EOF
 ----
 
-. If the `default` Ingress Controller configuration has the `spec.endpointPublishingStrategy: HostNetwork` value set, you must apply a label to the `default` {product-title} namespace to allow network traffic between the Ingress Controller and the project:
-
-.. Determine if your `default` Ingress Controller uses the `HostNetwork` endpoint publishing strategy:
-+
-[source,terminal]
-----
-$ oc get --namespace openshift-ingress-operator ingresscontrollers/default \
-  --output jsonpath='{.status.endpointPublishingStrategy.type}'
-----
-
-.. If the previous command reports the endpoint publishing strategy as `HostNetwork`, set a label on the `default` namespace:
-+
-[source,terminal]
-----
-$ oc label namespace default 'network.openshift.io/policy-group=ingress'
-----
-
-. Confirm that the `NetworkPolicy` object exists in your current project
-by running the following command:
-+
-[source,terminal]
-----
-$ oc get networkpolicy <policy-name> -o yaml
-----
-+
-In the following example, the `allow-from-openshift-ingress` `NetworkPolicy`
-object is displayed:
+. Optional: To confirm that the network policies exist in your current project, enter the following command:
```

Comment:
As the oc describe has namespace 'frontend-app', the text could say 'your current project (frontend-app)'

Reply:
@asood-rh, the header for the output is _.Example output_ so I think that it's okay that it might be different from what the user will see.